# MappingTaskMessage.msg
string mapName
string savePath